source/draw/gpu/GpuCanvasYuv420Semiplanar.ooc

```diff
@@ -36,14 +36,14 @@ GpuCanvasYuv420Semiplanar: class extends GpuCanvas {
 		set(value) {
 			this _pen = value
 			yuv := value color toYuv()
-			this _target y canvas pen = Pen new(ColorBgra new(yuv y, 0, 0, 255), value width)
-			this _target uv canvas pen = Pen new(ColorBgra new(yuv u, yuv v, 0, 255), value width)
+			this _target y canvas pen = Pen new(ColorRgba new(0, 0, yuv y, 255), value width)
+			this _target uv canvas pen = Pen new(ColorRgba new(0, yuv v, yuv u, 255), value width)
```

Comment:
I wonder if these are given with the default shader so that internal order is leaking.